tests/ruby/RuboCopBearTest.py

```diff
@@ -17,3 +19,24 @@
 RuboCopBearTest = verify_local_bear(RuboCopBear,
                                     invalid_files=(bad_file,),
                                     valid_files=(good_file,))
+
+rubocop_config = os.path.join(os.path.dirname(__file__),
+                              "test_files",
+                              "rubocop_config.yml")
+
+camelCaseFile = """
+def goodName
+  some content
+end
+""".splitlines(keepends=True)
+
+snake_case_file = """
+def bad_name
+  some content
+end
+""".splitlines(keepends=True)
```

Comment:
can you do the testcase with the previously defined files? Less duplication and you get proof that the config is changing behaviour, not the different content.